requirements.txt

```
@@ -1,2 +1,3 @@
 icalendar
 croniter
+recurrent
```

Wird croniter noch benötigt? Wenn nein, dann bitte auch das `import croniter` im events.py entfernen.

```suggestion
recurrent
```